Automattic/jetpack
# Backups: Add plugin list table link to Settings

Other Jetpack plugins have a "Settings | Deactivate" order and links from the plugin list table. This adds this to the Backup plugin.

#### Changes proposed in this Pull Request:

* Adds Settings like to the plugin list table.

#### Jetpack product discussion


n/a


#### Does this pull request change what data or activity we track or use?


n/a

#### Testing instructions:






* Install/activate plugin and visit the wp-admin Plugins list.
* Before patch, only see "Deactivate" as an action link. Afterwards, "Settings" is present.
